package.json

```diff
@@ -49,7 +49,7 @@
     "react-redux": "~4.4.5",
     "react-router": "~2.5.1",
     "react-router-redux": "~4.0.5",
-    "react-shortcuts": "~1.3.1",
+    "react-shortcuts": "git+https://github.com/Dunkat/react-shortcuts.git#dev",
```

Comment:
yup. added an issue to not forget: https://github.com/metasfresh/metasfresh-webui-frontend/issues/307